v3.0/how-to/deploy/data-migration-with-ansible.md

```diff
@@ -52,29 +56,37 @@ Make sure you have logged in to the Control Machine using the `root` user accoun
 
 1. Create the `tidb` user.
 
+    {{< copyable "shell-root" >}}
+
     ```
-    # useradd -m -d /home/tidb tidb
+    useradd -m -d /home/tidb tidb
     ```
 
 2. Set a password for the `tidb` user account.
 
+    {{< copyable "shell-root" >}}
+
     ```
-    # passwd tidb
+    passwd tidb
     ```
 
 3. Configure sudo without password for the `tidb` user account by adding `tidb ALL=(ALL) NOPASSWD: ALL` to the end of the sudo file:
 
+    {{< copyable "shell-root" >}}
+
     ```
-    # visudo
+    visudo &&
     tidb ALL=(ALL) NOPASSWD: ALL
```

Comment:
```suggestion
    tidb ALL=(ALL) NOPASSWD: ALL
```
This looks like the execution result. Please check the Chinese version to confirm it.